019_t_L54_safe.js

```
@@ -0,0 +1,361 @@
+      usernames[index].room = room;
+    }
+}
+
+// get's username for "tell". Originaly same used to get meme name
+function getUserName(message) {
+    message = message.slice(2); // remove "m " from beginning
+
+    if(message.indexOf(" ") != -1) var memeName = message.slice(0, message.indexOf(" ")); // get the meme name (if the text fields are not empty)
+
+    message = message.slice(message.indexOf(" ")+1); // remove the meme name
+    if(memeName === undefined) var memeName = message; // that's in case "m fwp", but messes up "m fwp tere".
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)